packages/react-core/src/components/Page/examples/Page.md

```diff
@@ -6,7 +6,7 @@ typescript: true
 propComponents: ['Page', 'PageHeader', 'PageSidebar', 'PageSection']
```

Comment:
Should the new Components be listed here so their props are included in the docs?